src/select-field.jsx

```diff
@@ -141,6 +141,8 @@ const SelectField = React.createClass({
       hintText,
       fullWidth,
       errorText,
+      onFocus,
```

Comment:
@oliviertassinari Yeah, that was my only concern, it's unclear if it affects existing behaviour.

I couldn't find much about whether `div` can accept focus, generally it seems it shouldn't but it's up to browser implementations.

Do you want me to remove the `onFocus`/`onBlur` from DropDownMenu as part of this PR or leave for wider review?